### PR TITLE
HA enabled ResourceManager compatible

### PR DESCRIPTION
**Pull request by xiaop1987**
1. When ResourceManager enabled ha, the older way can't get the resource manager correct.

2. When reusing the HttpConnection, the request will fail raising a ResponseNotReady exception.

### CODE REVIEW

**Comment by toidi** [comment]
Thanks for pull request @xiaop1987 

Can you fix tests?

**Comment by xiaop1987** [comment]
@toidi I've fixed the tests, but the coverage is good enough, i'll add some cases later

**Comment by toidi** [comment]
@xiaop1987 I think we can drop support to `python 3.2` and add tests for `python 3.4`, `python 3.5`.
It will fix tests.

**Comment by xiaop1987** [comment]
@toidi  All cases should pass now.

File: yarn_api_client/hadoop_conf.py [hunk 1]
```diff
@@ -1,21 +1,64 @@
...
-def get_resource_manager_host_port():
-    config_path = os.path.join(CONF_DIR, 'yarn-site.xml')
+def get_rm_ids(hadoop_conf_path):
+    rm_ids = parse(os.path.join(hadoop_conf_path, 'yarn-site.xml'), 'yarn.resourcemanager.ha.rm-ids')
+    if rm_ids is not None:
+        rm_ids = rm_ids.split(',')
+    return rm_ids
+
+
+def get_resource_manager(hadoop_conf_path, rm_id = None):
```
Comment (toidi) [inline yarn_api_client/hadoop_conf.py]: If `get_resource_manager` is a function for internal usage - prefix it with underscore.

**Comment by xiaop1987** [comment]
Hi @toidi , the method name issue you mentioned has been fixed, I add some cases to increase the coverage.

**Comment by toidi** [comment]
@xiaop1987 good job 👍 

Squash commits, bump version and I'll merge and update PyPI package.

**Comment by xiaop1987** [comment]
@toidi  I squashed the commits, but don't known did it correctly. Check it please.

**Comment by toidi** [comment]
@xiaop1987 sorry for the delay. I was quite busy last days.

I'll merge it tonight.

**Comment by xiaop1987** [comment]
@toidi no need to sorry at all, merge it whenever you are free :), your project here saved me lots of time, I'm very appreciate about that.

**Comment by toidi** [comment]
@xiaop1987 it's not what I expected. I'll create another PR with correct squashing

**Pull request closed without merging**